config_example.yml

```diff
@@ -42,11 +42,10 @@ home_alert_nodes:
     video_width: 1280
     video_height: 720
     # opencv_install may be one of
-    # apt: install opencv python3 bindings using apt, works with Raspbian Buster.
-    # stretch: Use a combination of apt and pip to get working python3 bindings on Raspbian Stretch.
-    # skip: you have installed opencv on your own, do not attempt any installation method.
+    # True: install opencv python3 bindings using apt, works with Raspbian Buster.
+    # False: you have installed opencv on your own, do not attempt any installation method.
     #
-    opencv_install: "skip"
+    opencv_install: False
```

Comment:
This should be true by default. In almost all circumstances do we want to install opencv via apt. Only in some edge  cases this might be false. So probably should explain in a comment here more clearly (like the comment added to the readme above).